server/sonar-server/src/main/java/org/sonar/server/property/InternalProperties.java

```diff
@@ -26,6 +26,11 @@
  * Allows to read and write internal properties.
  */
 public interface InternalProperties {
+  /**
+   * The UUID of the default organization.
+   * Can't be null unless SQ is strongly corrupted.
+   */
+  String DEFAULT_ORGANIZATION = "organization.default";
```

Comment:
what about mentioning "uuid" in the value ? For example "organization.defaultUuid"